spec/models/vote_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe Vote, type: :model do
+
+  it { should belong_to(:votable) }
+
+  it { should validate_presence_of :user_id }
+  it { should validate_presence_of :votable_type }
+  it { should validate_presence_of :votable_id }
+
```

Comment:
After adding the uniqueness validation to the model, be sure to cover it with a test here as well.